xtask/src/commands/changeset/mod.rs

```diff
@@ -0,0 +1,499 @@
+/// This module is generated:
+///   1. Only if you actually change the `matching_pull_request.graphql` query.
+///   1. By installing `graphql_client_cli`
+///
+///          cargo install graphql_client_cli
+///
+///   1. By running:
+///      1. A command that downloads the GitHub Schema.  It's large so we don't
+///         need to check it in.  Make sure to download it INTO the
+///         `src/commands/changeset/` directory.
+///
+///             wget https://docs.github.com/public/schema.docs.graphql`
+///
+///      2. Generate against this downloaded schema.  Run this from inside the
+///         `src/commands/changeset/` directory.
+///
+///             graphql-client generate \
+///               --schema-path ./schema.docs.graphql \
+///               --response-derives='Debug' \
+///               --custom-scalars-module='crate::commands::changeset::scalars' \
+///               ./matching_pull_request.graphql
+///
+mod matching_pull_request;
+mod scalars;
+
+use ::reqwest::Client;
+use anyhow::Result;
+use console::style;
+use dialoguer::{console::Term, theme::ColorfulTheme, Confirm, Editor, Input, Select};
+use git2;
+use itertools::Itertools;
+use matching_pull_request::matching_pull_request::Variables;
+use matching_pull_request::MatchingPullRequest;
+use memorable_wordlist;
+use serde::Serialize;
+use std::{fmt, fs, path::PathBuf, slice::Iter, str::FromStr};
+use structopt::StructOpt;
+use tinytemplate::{format_unescaped, TinyTemplate};
+use xtask::*;
```

Comment:
It looks like our rustfmt config is not applied in xtask’s own code, likely because it’s in a different workspace. But changing this is likely for another PR.

Reply:
Yeah, I definitely couldn't figure out what was up there but I think that whole subtree definitely needs a formatting run.

Reply:
Opened https://github.com/apollographql/router/issues/2557 to track this follow-up work.